### DISCUSSION TITLE
LPPL-1.3c license specificities

### DISCUSSION THREAD

**Discussion by GilOliveira** [joaomlourenco/novathesis#214] [Q&A] [answered]
Hello everyone!
I always struggle to understand legalese but I have a question regarding the requirements of the license used in this template.

Because novathesis is licensed under the LPPL-1.3c, does that mean that I am required to publicly disclose the source code I used to make my thesis?

Thanks a bunch and thank you all for the great work!

**Comment by b-pereira** [comment, marked as answer]
**tl:dr;** No. When the licence means **Work** is source code. WORK is not the "use" of the WORK.

Also [LPPL-1.3c](https://www.latex-project.org/lppl/lppl-1-3c/) is not GPL.

![imagem](https://user-images.githubusercontent.com/944511/131655024-b6a8581a-4ceb-414f-aa56-745c745d0517.png)

For the _rationale_:
[LPPL 1.3c: What is a "compiled work"?](https://tex.stackexchange.com/questions/602321/lppl-1-3c-what-is-a-compiled-work) where someone in the answers pointed to this [GPL-Licensed LaTeX template - implications for resulting work?](https://opensource.stackexchange.com/questions/2735/gpl-licensed-latex-template-implications-for-resulting-work). Thus, finally [Is a PDF output from a LaTeX document, a "derived work" from the LPPL standard packages? [closed]](https://tex.stackexchange.com/questions/82713/is-a-pdf-output-from-a-latex-document-a-derived-work-from-the-lppl-standard-p/82719#82719) where "The answer is clearly and unambiguously NO". If you're still not sure, see the second answer from this last link, **especially the replies**.

**Reply by b-pereira** [reply]
@GilOliveira don't forget to mark this as "Answered". Thank you.
